docs/contributor/arch/013-resource-attributes-enrichment-for-non-workload-metrics.md

```diff
@@ -27,14 +27,18 @@ There are different possible solutions to solve this problem:
 ![Connectors](../assets/connectors.drawio.svg)
 
 We can split our MetricPipeline into 3 sub-pipelines which are connected using [Connectors](https://opentelemetry.io/docs/collector/configuration/#connectors) as shown in the diagram above.
-An `Input Pipeline`, which contains the `otlp` receiver and `memory_limiter` processor.
-An `Attributes Enrichment Pipeline`, which contains the `k8sattributes` and `transform/resolve-service-name` processors.
-An `Output Pipeline`, which contains the rest of the processors and the `otlp` exporter.
+The `Input Pipeline` contains the `otlp` receiver and `memory_limiter` processor.
+The `Attributes Enrichment Pipeline` contains the `k8sattributes` and `transform/resolve-service-name` processors.
+The `Output Pipeline` contains the rest of the processors and the `otlp` exporter.
 
-The metrics will be routed to the `Attributes Enrichment Pipeline` only if they are workload metrics. This can be done by checking if a resource attribute (like `skip-enrichment`) is set `true`.
-Otherwise, the metrics will bypass the `Attributes Enrichment Pipeline` and will be routed directly to the `Output Pipeline`.
+If the resource attribute `skip-enrichment` is set `true`, the `Routing Connector` will route the metrics to the `Output Pipeline` directly. Otherwise, the `Routing Connector` will route the metrics to the `Attributes Enrichment Pipeline`.
```

Comment:
```suggestion
If the resource attribute `skip-enrichment` is set to `true`, the `Routing Connector` will route the metrics to the `Output Pipeline` directly. Otherwise, the `Routing Connector` will route the metrics to the `Attributes Enrichment Pipeline`.
```